desc/coils.py

```diff
@@ -1396,27 +1422,72 @@ def _compute_position(self, params=None, grid=None, **kwargs):
             )
             xyz_sym = xyz @ reflection_matrix(normal).T @ reflection_matrix([0, 0, 1]).T
             xyz = jnp.vstack((xyz, jnp.flipud(xyz_sym)))
+            if dx1:
```

Comment:
You test it on a `MixedCoilSet`, which is never symmetric by construction, but it contains a symmetric `CoilSet` so I think these lines are supposed to be covered by the tests. This makes me think that the `dx1` argument is not being passed in correctly to the sub-coils of the MixedCoilSet

Reply:
Should be fixed now. The weirdness is because `CoilSet.from_symmetry` returns something with `sym=False` and `NFP=1`, because it doesn't use "virtual" coils, it manually duplicates and rotates the input coils. which makes me wonder if we should get rid of that? Since you get the same thing by just doing `CoilSet(coils, NFP=5, sym=True)` but its more efficient? Or maybe we just rename it to something like `CoilSet.materialize_virtual_coils` or something...

Reply:
I think we can leave it as it. I believe the original design of `CoilSet.from_symmetry` was to be an easy way to initialize coils, but yes in general you shouldn't use it for symmetry. It could still be useful if you override the default to only rotate through a half field period, then create a symmetric `CoilSet` from those unique coils.